org.gnome.Showtime.json

```diff
@@ -0,0 +1,91 @@
+{
+  "id": "org.gnome.Showtime",
```

Comment:
is this an official GNOME app?

Reply:
Yes, it is in [Incubator](https://gitlab.gnome.org/GNOME/Incubator).